Library/Homebrew/test/testing_env.rb

```diff
@@ -26,6 +26,10 @@ def version(v)
       Version.new(v)
     end
 
+    def head_version(v)
+      HeadVersion.new(v)
+    end
```

Comment:
Can we avoid adding this method and instead modify the `version` method directly above to use `Version.create`?